dubbo-common/src/main/java/org/apache/dubbo/common/utils/PojoUtils.java

```diff
@@ -774,7 +778,8 @@ private static Field getField(Class<?> cls, String fieldName) {
         for (Class<?> acls = cls; acls != null; acls = acls.getSuperclass()) {
             try {
                 result = acls.getDeclaredField(fieldName);
-                if (!Modifier.isPublic(result.getModifiers())) {
+                // the field is not public and it not jdk class, we will setAccessible to true
+                if (!Modifier.isPublic(result.getModifiers()) && acls.getClassLoader() != null) {
```

Comment:
For JDK internal classes, even if they don't fail here, they will eventually fail.